### DISCUSSION TITLE
Voice Clips

### DISCUSSION THREAD

**Discussion by UltramanTime** [Techokami/SonicWorldsNext#52] [Ideas]
Here's my another idea for Sonic World Next:

All Characters should have Voice Clips:

Ryan Drummond for Sonic the Hedgehog
Corey Bringas for Tails
Michael McGaharn for Knuckles the Echidna (Oh No)
Jennifer Douillard for Amy Rose
David Humphrey for Shadow the Hedgehog

**Comment by BlipShift2005** [comment]
You can make a fork of the source code and add it yourself.

**Comment by Techokami** [comment]
Character voice clips are outside of the scope of this project...

**Comment by naoshibaikyu** [comment]
The entire point of frameworks is that you build a game with them (or make your own fork of this framework) and do these additions yourself. Asking for extra things like voice acting does not make sense for a framework like this, especially one that's based on the 16-bit styled gameplay.

Please knock it off.
